drush_cmi_tools.drush.inc

```diff
@@ -87,24 +88,40 @@ function drush_drush_cmi_tools_config_export_plus($destination = NULL) {
       catch (InvalidDataTypeException $e) {
         $ignore_list_error = TRUE;
       }
-      if (!isset($parsed['ignore']) || !is_array($parsed['ignore'])) {
+      if (!isset($parsed['ignore']) || !is_array($parsed['ignore']) || !isset($parsed['ignore_collection']) || !is_array($parsed['ignore_collection'])) {
```

Comment:
this is a hard break - can we make existing config-ignore.yml files that don't have the ignore_collection key continue to work?

e.g. we could 

```php
$parsed += ['ignore_collection' => []];
``